lib/lib.nix

```diff
@@ -210,9 +206,14 @@
     nickelResult = callNickel {
       inherit nickelFile baseDir flakeInputs lockFileContents;
     };
+    enrichedFlakeInputs =
+      flakeInputs
+      // {
+        "%%organist_internal".nickelLock = builtins.toFile "nickel.lock.ncl" (buildLockFileContents lockFileContents);
```

Comment:
Could be nice to add a little comment explaining why we need to do that, for future you, me and others

Reply:
Indeed. Fixed

Reply:
Has the doc been lost in a force push?I can't see it still